docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md

```diff
@@ -0,0 +1,189 @@
+      On-premises
+    </a>
+  </li>
+  <li>
+    <a href="#aws" class="nav-link" id="aws-tab" data-toggle="tab"
+      role="tab" aria-controls="aws" aria-selected="false">
+      <i class="fa-brands fa-aws"></i>
+      AWS
+    </a>
+  </li>
+  <li>
+    <a href="#gcp" class="nav-link" id="gcp-tab" data-toggle="tab"
+      role="tab" aria-controls="gcp" aria-selected="false">
+      <i class="fa-brands fa-google"></i>
+      GCP
+    </a>
+  </li>
+  <li>
+    <a href="#azure" class="nav-link" id="azure-tab" data-toggle="tab"
+      role="tab" aria-controls="azure" aria-selected="false">
+      <i class="fa-brands fa-microsoft"></i>
+      Azure
+    </a>
+  </li>
+  <li>
+    <a href="#k8s" class="nav-link" id="k8s-tab" data-toggle="tab"
+      role="tab" aria-controls="k8s" aria-selected="false">
+      <i class="fa-regular fa-dharmachakra"></i>
+      Kubernetes
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="onprem" class="tab-pane fade show active" role="tabpanel" aria-labelledby="onprem-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+
+It is recommended to use separate disks for the Linux OS and for the data.
+
+  </div>
+
+  <div id="aws" class="tab-pane fade" role="tabpanel" aria-labelledby="aws-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+- [Amazon Web Services](../../../deploy/checklist/#amazon-web-services-aws)
+
+Please note that ephemeral disk based instances like i3 that are recommended in this list come with certain restrictions on Day 2 YBA operations, so we typically recommend instances with EBS attached storage instead.
+
+  </div>
+
+  <div id="gcp" class="tab-pane fade" role="tabpanel" aria-labelledby="gcp-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+- [Google Cloud](../../../deploy/checklist/#google-cloud)
+
+Using local disks comes with certain restrictions on Day 2 YBA operations, so we typically recommend using remote disks instead.
+
+## GCP instance templates
```

Comment:
This doesn't seem to belong in the hardware requirements section because we don't require anything here?